cilium/cmd/service_update.go

```diff
@@ -28,6 +28,7 @@ var (
 	frontend           string
 	backends           []string
 	backendStates      []string
+	backendWeights     []uint
```

Comment:
Should this be defined as `uint16` to avoid type casting below?

Reply:
it's not that simple as `spf13/pflags` - `UintSliceVarP` support only `uint` :/.